app/src/androidTest/java/org/projectbuendia/client/ui/dialogs/EditPatientDialogFragmentTest.java

```diff
@@ -59,29 +59,36 @@ public void testNewPatient() {
 
         // The admission date should be visible right after adding a patient.
         // Flaky because of potential periodic syncs.
-        expectVisibleWithin(100000, viewThat(
+        expectVisibleWithin(399999, viewThat(
```

Comment:
Does this change actually cause a failing test to pass?  I'd be surprised to find that this usually took more than 100 seconds but less than 399 seconds.